src/language-css/utils.js

```diff
@@ -211,6 +211,14 @@ function hasParensAroundValueNode(node) {
   );
 }
 
+function isPostcssSimpleVar(currentNode, nextNode) {
```

Comment:
The check in here is a bit hard to understand. At a glance it feels too generic. Is it? Could his have unintended consequences? For example, do we need to check that the name of something is `$`?

Reply:
/cc @ad1992

Reply:
Perhaps the function logic is fine, but the name `isPostcssSimpleVar` is misleading? Because that's not what it checks?

Reply:
initially the code was
```js
iNode.type === "value-func" &&
hasNoSpaceBeforeValueNode(iNextNode)
```
and then changed it to isPostcssSimpleVar. Yes its not explicitly checking whether is `PostcssSimpleVar`. 
should it be isLeadingSpaceInPostCssSimpleVar ? Please suggest some better name :)

Reply:
@ad1992 just add check on `$` in your utils function and all